WWCodeTokyo/speak-her-db
# Switch name to name_ja on ja locale

Resolves #121 

Use the Japanese name when locale is 'ja'.

**Testing plan:**

1. Load the main page and scroll down to view the speakers
2. Toggle the locale to Japanese
3. Check that the speakers' names are in Japanese
4. Toggle locale back to English
5. Check that the speakers' names are in English
